.github/workflows/build_changed_packages.yml

```diff
@@ -1,4 +1,4 @@
-name: Build changed packages
+name: Build
```

Comment:
Why don't you change the file name as well?

Reply:
Oh~ I will changed the file name!